ui/css/itcss/components/send.scss

```diff
@@ -864,7 +864,7 @@
     border: 1px solid var(--color-border-muted);
     border-radius: 4px;
     background-color: var(--color-background-default);
-    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.14);
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
     display: flex;
     flex-flow: column;
 
```

Comment:
Pretty sure this is all unused CSS that should be removed. Will create a new PR and remove it.